server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -35,7 +35,25 @@ export class Throttler implements IThrottler {
             max: maxCacheSize,
             maxAge: maxCacheAge,
         };
-        this.lastThrottleUpdateAtMap = new LRUCache(cacheOptions);
+        this.lastThrottleUpdateAtMap = new LRUCache({
+            ...cacheOptions,
+            dispose: (key, value: number) => {
```

Comment:
@arikt-ms - since you recently worked on a similar LRU cache on the AFR side, I thought you might find this interesting. I can't remember now if that implementation already includes telemetry on things like eviction

Reply:
@hedasilv right, this is interesting feature that might be helpful in debugging issues I think. I have not used this in my implementation but we definitely can add it later if we need to.